aboutme.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Portfolio Liliya Voevodina.html</title>
+        <link href="css/normalize.css" rel="stylesheet" type="text/css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link href="css/stylesheet.css" rel="stylesheet" type="text/css">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <div class="aboutme-hometop">
+            <nav>
+                <img class="aboutme-mypicture" src="images/IMG_9169.jpg" alt="Liliya">
+                <a href="#">Contacts</a>
+                <a href="#">Portfolio</a>
+                <a href="aboutme.html">About me</a>
+                <a href="homepage.html">Home</a>
+            </nav>
+            <div class="aboutme">
+            <h3>
+                I wish i was a developer...but i am not.
+            </h3>
+            <h4>
+                <p>
+                    YET.
+                </p>
+                <p>
+                    I am a full stack student of a Mayden Academy in Bath, UK
+                </p>
+                <p>
+                    Academy so rapidly takes me to a moment, where i will be able to proudly
+                    call myself a developer, that by the end of my course this text will be already different :)
+
+                </p>
+
+            </h4>
+            </div>
+        </div>
+        <div class="homecenter">
+            <img class="aboutme-arrows" src="images/arrow-img.png" alt="Arrows">
+            <div class="aboutme-social">
```

Comment:
why do you have different classes for the about me page? this should be the same as the homepage?